src/extensions/extension-loader/extension-loader.ts

```diff
@@ -270,11 +270,12 @@ export class ExtensionLoader {
     });
   };
 
-  loadOnClusterRenderer = (entity: KubernetesCluster) => {
+  loadOnClusterRenderer(getCluster: () => KubernetesCluster) {
```

Comment:
@panuhorsmalahti notice that arrow function here is important.

Reply:
Fixed (i.e. reverted change, which was due to rebasing error).